### ISSUE TITLE
Prescript does not work

### ISSUE THREAD

**Issue by nbittmann** [Jalle19/node-ffmpeg-mpegts-proxy#19]
@Jalle19 
I'm trying to use the rescript option to refresh a session ID before starting the transcoding however this does not seem to work.

`
[
    {
        "url": "/channel1",
        "source": "http://channel1.example"
        "prescript": "/usr/bin/python Update.py channel1 Channel 1",
        "name": "Channel 1",
        "provider": "channel1"
    },
]
`

The script is written in python and works just fine if executed manually.I've also tried full paths but no success also the log does not output anything at all.

Any ideas?

Thanks

**Comment by nbittmann** [comment]
Appears it does not work if you pass arguments to it, found an alternative way though.

**Comment by Jalle19** [comment]
The application automatically passes the stream URL as the first parameter to script. Which parameters do you need? I guess we could just pass all relevant parameters, then it's up to the script to use the ones it needs.

**Comment by nbittmann** [comment]
@Jalle19 

Thanks for coming back, I've got a python script taking two parameters which is a provider ID and a Channel name unfortunately these are not represented in the URL and the purpose of this script is to update the URL before starting the video in order to get a new session ID.

**Comment by Jalle19** [comment]
I can change the code so that it passes the provider, the channel name and the URL instead of just the URL, would that help?

**Comment by nbittmann** [comment]
Hi,

Yes that would help I could then adapt my script to use these parameters !

Thanks a lot !

**Comment by Jalle19** [comment]
Closing this in favor of https://github.com/Jalle19/node-ffmpeg-mpegts-proxy/issues/26
